examples/fpga/artya7/README.md

```diff
@@ -11,8 +11,19 @@ Please see [examples](https://ibex-core.readthedocs.io/en/latest/examples.html "
   - `fusesoc` and its dependencies
   - Xilinx Vivado
 
+### Hardware
+
+  - Either a Digilent Arty A7-35 oder A7-100 board
+
 ## Build
 
+The easiest way to build and execute this example is to call the following make goals that match your hardware from the root directory.
+
+  - `build-arty-35`
+  - `program-arty-35`
+  - `build-arty-100`
+  - `program-arty-100`
```

Comment:
It might be nicer to have the following:

Use the following for the Arty A7-35
````
make build-arty-35 program-arty-35
````
and for the Arty A7-100
````
make build-arty-100 program-arty-100
````